### PR TITLE
Feature/display authors on examples

### PR DESCRIPTION
**Pull request by ClementPasteau**
Similar to displaying Authors on Extensions list & details page. Now for Examples.

Screenshot 2021-09-24 at 11 09 47
Screenshot 2021-09-24 at 11 08 53

### CODE REVIEW

File: newIDE/app/src/ResourcesList/index.js [hunk 1]
```diff
@@ -12,7 +12,6 @@ import { showWarningBox } from '../UI/Messages/MessageBox';
 import { filterResourcesList } from './EnumerateResources';
 import { mapVector } from '../Utils/MapFor';
 import optionalRequire from '../Utils/OptionalRequire.js';
-import Window from '../Utils/Window';
```
Comment (4ian) [inline newIDE/app/src/ResourcesList/index.js:15]: Thanks for cleaning after me!

**Review by 4ian — APPROVED** [review]
Looks like a straightforward addition, as we could have been hoping, LGTM!

**Comment by ClementPasteau** [comment]
Few additions to ensure we are not using the old way of fetching Examples (use directly the object returned rather than fetching the json again) :)

File: newIDE/app/src/Utils/GDevelopServices/Example.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  }
+  const filtersResponse = await axios.get(filtersUrl);
+
+  return {
+    exampleShortHeaders: examplesDatabase,
+    filters: filtersResponse.data,
+  };
+};
+
+export const getExample = (
```
Comment (4ian) [inline newIDE/app/src/Utils/GDevelopServices/Example.js:47]: You can as well make this async?

**Review by 4ian — APPROVED** [review]

**Pull request merged**